src/app/pages/products-list/card/card.component.html

```diff
@@ -1,23 +1,19 @@
 <mat-card class="card">
     <mat-card-header>
-        <mat-card-title>9.6" Планшет Dexp Ursus S290 32 ГБ 3G черный</mat-card-title>
-        <mat-card-subtitle>86 $</mat-card-subtitle>
+        <mat-card-title>{{ product?.name }}</mat-card-title>
+        <mat-card-subtitle>{{ product?.price | currency }}</mat-card-subtitle>
     </mat-card-header>
     <div class="galerey">
         <button mat-icon-button disabled>
             <mat-icon>chevron_left</mat-icon>
         </button>
-        <img
-            mat-card-image
-            alt="Product photo"
-            src="https://c.dns-shop.ru/thumb/st1/fit/0/0/dff090e667cd39dd397678905df52de5/6c3e7aab802deb19513c7cf55dd7c527efb9127d8aba2ed90f3d224577137fe2.jpg"
-        />
+        <img mat-card-image [alt]="product?.name" [src]="product && product.images && product.images[0]?.url" />
```

Comment:
Рекомендация: тут можно было бы использовать просто цепочку `?.`
```suggestion
        <img mat-card-image [alt]="product?.name" [src]="product?.images?.[0]?.url" />
```